packages/components/psammead-program-card/src/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import styled from 'styled-components';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import Paragraph from '@bbc/psammead-paragraph';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  C_EBON,
+  C_METAL,
+  C_SHADOW,
+  C_WHITE,
+  C_POSTBOX,
+  C_KINGFISHER,
+} from '@bbc/psammead-styles/colours';
+import {
+  getSansRegular,
+  getSerifRegular,
+  getSansBold,
+} from '@bbc/psammead-styles/font-styles';
+import {
+  getLongPrimer,
+  getMinion,
+  getPica,
+} from '@bbc/gel-foundations/typography';
+import { Link } from '@bbc/psammead-story-promo';
+
+const programStateColors = {
```

Comment:
Is it worth defining these states as constants and then exporting them from this package too? So then we can make sure we're passing in a valid state that matches what we expect in here.

Reply:
That approach makes sense, we can export them from this package for now. We might need to move them to a utility file in the future.

Reply:
Thinking about it again, we could just use `oneOf` in propTypes to make sure we pass in a valid state!